products/redis/api.yaml

```diff
@@ -15,13 +15,13 @@
 name: Redis
 display_name: Cloud Memorystore
 versions:
```

Comment:
We're now assuming that versions will have a given ordering. Could we some combination of:

1) Sort the versions in MM so that the order doesn't matter in the yaml files
2) Document this order somewhere.
3) Have a check somewhere in code that complains if `api.yaml` is in an incorrect order

Reply:
Alright, looking back through `default_version`, it looks like the ordering isn't forced. Is that correct?

Reply:
Ordering should not be forced. If we compile at a specific version IE `alpha` it will look up the ordering and fall back to the closest version in order that the object exists on.

So compiling at `alpha` will first look if there is an alpha version, then beta, then ga

Reply:
yeah I'm pretty sure the order is set https://github.com/GoogleCloudPlatform/magic-modules/blob/master/api/product/version.rb#L29